src/System.IO.Compression/tests/ZipArchive/zip_ReadTests.cs

```diff
@@ -185,5 +184,16 @@ public class zip_ReadTests : ZipFileTestBase
 
             s.Dispose();
         }
+
+        [Fact]
+        public static async Task Crc32Test()
+        {
+            ZipArchive archive1 = new ZipArchive(await StreamHelpers.CreateTempCopyStream(zfile("normal.zip")), ZipArchiveMode.Read);
+            ZipArchive archive2 = new ZipArchive(await StreamHelpers.CreateTempCopyStream(zfile("normal.zip")), ZipArchiveMode.Read);
+            ZipArchive archive3 = new ZipArchive(await StreamHelpers.CreateTempCopyStream(zfile("small.zip")), ZipArchiveMode.Read);
+
+            Assert.True(archive1.Entries[0].Crc32 == archive2.Entries[0].Crc32);
```

Comment:
Crc32 is only exposed in netcoreapp, but these tests also run for other configurations, like against the .NET Framework, where this property doesn't yet exist. The test needs to be added in a .netcoreapp.cs specific file.  You can see other examples in this project by looking at the .csproj.